projects/igniteui-angular/src/lib/grids/grid/row-pinning.spec.ts

```diff
@@ -237,3 +325,36 @@ export class GridRowPinningComponent {
     @ViewChild(IgxGridComponent, { read: IgxGridComponent, static: true })
     public instance: IgxGridComponent;
 }
+
+@Component({
+    template: `
+    <igx-grid [data]="data" height="500px" [pinning]='pinningConfig'>
+        <igx-column-group header="General Information">
```

Comment:
Column groups are not supported with MRL. [There should be no columns outside of those blocks and no usage of IgxColumnGroupComponent when configuring a multi row layout.](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/multi_row_layout.html)